### DISCUSSION TITLE
Display Server Info on dashboard.rb

### DISCUSSION THREAD

**Discussion by TeamRainless** [activeadmin/activeadmin#8054] [Help] [answered]
As the question asks... Is there a way to get dashboard.rb to display the current server info? (Rails version, ruby version, etc...)

Somewhere around here:
  ```
# Here is an example of a simple dashboard with columns and panels.
    #
    # columns do
    #   column do
    #     panel "Info" do
    #       para "Welcome to ActiveAdmin."
    #     end
    #   end
    # end
```

**Comment by tagliala** [comment, marked as answer]
A possible approach would be:

```rb
ActiveAdmin.register_page 'Dashboard' do
  # ...

  content title: proc { I18n.t('active_admin.dashboard') } do
    panel 'Info' do
      span "Rails Version:"
      b Rails.version
      span " - Ruby Version:"
      b RUBY_VERSION
      span " - Active Admin Version:"
      b ActiveAdmin::VERSION
    end
  end
end
```

**Comment by TeamRainless** [comment]
"Resolved" as in "fixed"? Or as in "the workaround is the way to go"?

**Reply by tagliala** [reply]
Hello, yes, this is the way to go

However, at the time there wasn't Active Admin v4. The current master branch and the v4 beta are customizable with custom templates

Ref: https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md#from-v3-to-v4-beta

**Comment by TeamRainless** [comment]
Okay... so when it says: "there is *no sortable functionality for has-many
forms*"

...the workaround should be effective in 4.0?

**Reply by tagliala** [reply]
Sorry, I don't get this question

If you need that functionality, you can stay on 3.x and use the answer marked as correct

If you don't need that functionality, you can upgrade to 4.x and customize the `footer` partial according to your needs
